components/brave_wallet/browser/json_rpc_service_unittest.cc

```diff
@@ -760,7 +760,7 @@ class JsonRpcServiceUnitTest : public testing::Test {
                                   const std::string& name,
                                   const std::string& decimals) {
     url_loader_factory_.SetInterceptor(base::BindLambdaForTesting(
-        [&, network_url, chain_id](const network::ResourceRequest& request) {
+        [=, this](const network::ResourceRequest& request) {
           std::string_view request_string(request.request_body->elements()
```

Comment:
symbol, name and decimals were captured by ref and didn't exist at callback's run time.
@onyb fyi